packages/discovery-provider/src/tasks/index_payment_router.py

```diff
@@ -182,11 +197,17 @@ def get_tx_in_db(session: Session, tx_sig: str) -> bool:
     return exists
 
 
-def get_purchase_metadata_from_memo(
+def parse_route_transaction_memo(
     session: Session, memos: List[str], timestamp: datetime
-) -> Union[PurchaseMetadataDict, None]:
-    """Checks the list of memos for one matching the format of a purchase's content_metadata, and then uses that content_metadata to find the stream_conditions associated with that content to get the price"""
+) -> RouteTransactionMemo:
+    """Checks the list of memos for one matching a format of a purchase's content_metadata, and then uses that content_metadata to find the stream_conditions associated with that content to get the price"""
```

Comment:
surprise lint doesn't catch this long string

Reply:
Fun fact: when the correct linting settings don't get picked up, I get yelled at. But I think our overrides in the project config allow for longer lines.